adapters/ddf-adapter/src/test/java/com/connexta/replication/adapters/ddf/DdfNodeAdapterTest.java

```diff
@@ -316,7 +316,110 @@ private Resource getResource() {
     Response response = mock(Response.class);
     when(status.getFamily()).thenReturn(failure ? Family.SERVER_ERROR : Family.SUCCESSFUL);
     when(response.getStatusInfo()).thenReturn(status);
-    when(function.apply(resource)).thenReturn(response);
+    when(function.apply(any())).thenReturn(response);
     return resource;
   }
+
+  private static final String RAW_METADATA =
```

Comment:
Since we're using a static method to marshal/unmarshal metadata in the adapter we need to use real metadata to avoid errors.